docs/resources/Channel.md

```diff
@@ -6,31 +6,38 @@ Represents a guild or DM channel within Discord.
+| recipients?            | array of [user](#DOCS_RESOURCES_USER/user-object) objects                  | the recipients of the DM                                                                                                                                                        |
+| icon?                  | ?string                                                                    | icon hash                                                                                                                                                                       |
+| owner_id?              | snowflake                                                                  | id of the DM or creator                                                                                                                                                         |
+| application_id?        | snowflake                                                                  | application id of the group DM creator if it is bot-created                                                                                                                     |
+| parent_id?             | ?snowflake                                                                 | for guild channels: id of the parent category for a channel (each parent category can contain up to 50 channels), for threads: id of the text channel this thread was created   |
+| last_pin_timestamp?    | ?ISO8601 timestamp                                                         | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                      |
+| rtc_region?            | ?string                                                                    | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the voice channel, automatic when set to null                                                                  |
+| video_quality_mode?    | integer                                                                    | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel, 1 when not present                                            |
+| message_count?         | integer                                                                    | an approximate count of messages in a thread, stops counting at 50                                                                                                              |
+| member_count?          | integer                                                                    | an approximate count of users in a thread, stops counting at 50                                                                                                                 |
```

Comment:
Since `thread_metadata` is being introduced, is there a reason that these properties shouldn't be in the metadata collection too as they're thread specific?